### ISSUE TITLE
GR alpha broken on Juno in v0.25.2?

### ISSUE THREAD

**Issue by dgleich** [JuliaPlots/Plots.jl#2078]
I'm going to dig into this one more and may eventually close this issue. 

After updating to v0.25.2, the following plot no longer had alpha-blended edges. (This doesn't include data, but I don't have time to check other plots right now.)

```
using GraphRecipes
gr()
graphplot(ei, ej, x =xy[:,1], y=xy[:,2],
  markercolor=:black, markerstrokecolor=:white,
  size=(1200,1200),dpi=300,
  markersize=0, linecolor=1, linealpha=0.01, linewidth=0.5,
  markeralpha=0.2,colorbar=false,
  axis_buffer=0.02, background=nothing)
```

I know this isn't a fully working example. I will work to get one, but I have a few other deadlines. However, as I didn't see any issue that seemed relevant, I wanted to post it. 

I fixed my issue by downgrading to v0.25.1

`] pin Plots@0.25.1`

Then the plot worked again.

julia> versioninfo()
Julia Version 1.1.0
Commit 80516ca202 (2019-01-21 21:24 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-1650 v3 @ 3.50GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)
Environment:
  JULIA_NUM_THREADS = 6

**Comment by daschw** [comment]
Thanks for reporting @dgleich!
Here's a MWE: `using Plots; plot(1:2, linealpha = 0.1)`.
I'll look into it.

**Comment by dgleich** [comment]
Thanks for looking into it and dealing with the incomplete report!

**Comment by daschw** [comment]
It will be fixed soon via #2080
